docker/tensorflow/gpu/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu20.04
```

Comment:
What should I really use?
`nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu20.04` or `nvidia/cuda:11.0.3-cudnn8-devel-ubuntu20.04`
For more information please see _Overview of Images_ section [here](https://hub.docker.com/r/nvidia/cuda). My opinion: `runtime` is enough.